### DISCUSSION TITLE
Is there a way to specify a WandB entity to which runs will be sent?

### DISCUSSION THREAD

**Discussion by vrn-sn** [PKU-Alignment/omnisafe#303] [Ideas]
I was going to make a feature request for this, but I wanted to make sure it doesn't already exist in OmniSafe.

There is a `wandb_project` variable that can be set, but it doesn't look like there's a `wandb_entity` or support for something similar. My team members have individually changed their default entity (in WandB) to the team entity to ensure that each person's runs get added to our team's projects, but this isn't ideal for teams that might want to send results to different entities (e.g. if contributing to projects on multiple teams).

**Comment by Gaiejj** [comment]
Yes, OmniSafe currently does not support specifying the wandb entity. Your team could refer to the parameter passing related to wandb in `omnisafe/common/logger.py`. I believe you can: 
- add the `wandb_entity` parameter to the `logger_config` section of the corresponding algorithm's YAML file
- and make the necessary modifications in `logger.py` to implement this. 

Thank you for your team's discovery; we believe it is really helpful.
